### PR TITLE
Add List.html

### PR DESCRIPTION
**Pull request by pourmand1376**
It is still not finished. I just wanted to see what you think about it. 

I've also committed to a branch in your repo to allow you to add some changes to it. So feel free if you want to change anything in my PR.

### CODE REVIEW

**Comment by pourmand1376** [comment]
Wow, This bot is amazing. Thanks for adding it.

**Comment by rohandebsarkar** [comment]
I am making some changes to this PR. Please do not merge until then.

File: theme.yaml [hunk 1]
```diff
@@ -21,3 +21,6 @@ original:
   name: al-folio
   homepage: https://alshedivat.github.io/al-folio/
   repo: https://github.com/alshedivat/al-folio
+
+params:
+  recent_posts_count: 3
```
Comment (rohandebsarkar) [inline theme.yaml:26]: Do you have a reference for this? I can't find any. And where is it used?

File: config/_default/menu.yaml [hunk 2]
```diff
@@ -1,10 +1,12 @@
 main:
-- identifier: home
-  name: Home
+- name: Home
```
Comment (rohandebsarkar) [inline config/_default/menu.yaml:2]: The title link on the left of the navbar also points to the home page. Therefore is it required to give another home link?

Comment (pourmand1376) [reply config/_default/menu.yaml:2]: It is a matter of personal preference. I like to have a home page just like [my website](https://amirpourmand.ir/). 

We can comment this or make it by default. I leave it to you.

File: theme.yaml (hunk 1 above)
Comment (pourmand1376) [reply theme.yaml:26]: No where. I had an idea which is not fully implemented. Let's delete it for now.

Comment (pourmand1376) [reply theme.yaml]: I deleted this one and added a new variable in `params.yaml`. It shows recent blogs.

**Comment by pourmand1376** [comment]
I also added recent-blogs. The only problem is that I can't make these css right. Could you fix it please?

**Comment by rohandebsarkar** [comment]
> I also added recent-blogs. The only problem is that I can't make these css right. Could you fix it please?

Sure, I will look into the CSS a bit later.

**Comment by pourmand1376** [comment]
I also added `katex` for math typesetting.

**Comment by pourmand1376** [comment]
Let's just close this PR as soon as you can. Lots of things depend on this PR and I really can't submit another PR since changes are not incorporated into master. 

I added two options for selecting if a user want to has social icons in navbar or index or not.

**Comment by rohandebsarkar** [comment]
Done from my side. The blog page looks nice now. Thanks for adding KaTeX I was thinking to try it out, as it seemed interesting, its a lot faster than MathJax.

**Comment by rohandebsarkar** [comment]
This looks good. The posts will need to be updated to be more through, that can be done in a later PR.

Merging!

**Pull request merged**